docker-compose.yml

```
@@ -0,0 +1,25 @@
+version: '3'
+services:
+  shopware-local:
+    image: dockware/dev:6.4.18.0
```

suggestion: Should we use the "latest" tag instead of a specific version?
```suggestion
    image: dockware/dev:latest
```

---

There is issue with latest, once shopware 7 will be released, from dockware we will get image with shopware7